ml/automl/tasks/model_training.py

```
@@ -0,0 +1,683 @@
+            "message": f"Training failed: {str(e)}",
+            "model_path": None,
+            "centroids_path": None,
+            "training_duration": 0,
+            "final_epoch": 0
+        }
+
+
+@task(name="validate_trained_model")
+def validate_trained_model(
+    model_path: str,
+    centroids_path: str,
```

Ruff (0.14.3)

Unused function argument: `centroids_path`

(ARG001)